josemak25/hairx-client
# Ft hair care screen

#### What does this PR do?

Have the hair care screen feature of the app working.

#### Description of Task to be completed?

Have the hair care screen show the following:

1. a logo.
2. Hair profile
3. message for when no list of visited saloon is empty 

#### How should this be manually tested?

After cloning the repo,
`CD` into it and `RUN`

javascript
git checkout develop

git pull origin develop

yarn install

yarn test

Using an emulator: Launch the app, and you should see the hair care screen after loading.

#### Screenshots (if appropriate)
![Simulator Screen Shot - iPhone Xs - 2020-04-24 at 15 03 00](https://user-images.githubusercontent.com/12932317/80239170-a367ff80-8657-11ea-8719-62be173014c6.png)
